server/utils/applications/utils.ts

```diff
@@ -171,12 +174,27 @@ const getApplicationType = (application: Application): ApplicationType => {
   return 'Standard'
 }
 
+const mapTimelineEventsForUi = (timelineEvents: Array<TimelineEvent>): Array<UiTimelineEvent> => {
+  return timelineEvents
+    .sort((a, b) => Number(DateFormats.isoToDateObj(b.occurredAt)) - Number(DateFormats.isoToDateObj(a.occurredAt)))
+    .map(timelineEvent => {
+      return {
+        label: { text: sentenceCase(timelineEvent.type) },
```

Comment:
I think it'd be better to have a translation here, as the Approved Premises prefix isn't relevant to the user

Reply:
yeah good shout